feat: added an example react app with API usage (VF-2072)

davidjhlu opened the pull request:
<!-- You can erase any parts of this template not applicable to your Pull Request. -->

**Fixes or implements VF-2072**

### Brief description. What is this change?
Created an example react app to show voiceflow API usage

Commit:
feat: added an example react app with API usage (VF-2072)

matheuspoleza commented on react/App.js:
```diff
@@ -0,0 +1,60 @@
+
+
+const App = props => {
+  const apiKey = 'YOUR_API_KEY_HERE'; // change this to YOUR API key, it should look like this: VF.DM.XXXXXXX.XXXXXX... keep this a secret!
```
it's better to define this variable outside the App component

matheuspoleza commented on react/App.js:
```diff
@@ -0,0 +1,60 @@
+  const [message, setMsg] = useState("");
+  const [resArr, updateArr] = useState([]);
+  const handleClick = async() => {
+    let data = await fetch(`https://general-runtime.voiceflow.com/`, { // change this to whatever your API URL is
```
put this `https://general-runtime.voiceflow.com/` to a variable outside the component, could be called `RUNTIME_API_URL`

matheuspoleza commented on react/App.js:
```diff
@@ -0,0 +1,60 @@
+      }
+    }
+    console.log(message);
+    console.log(user);
```
do we really need those `console.log` in the browser example?

matheuspoleza commented on react/App.js:
```diff
@@ -0,0 +1,60 @@
+  }
+
+
+  const handleChangeUser = (event) => {
```
💅 but don't need the `{}` here. same for the other handler

``` 
const handleChangeUser = (event) => setUser(event.target.value);
```

matheuspoleza commented on react/App.js:
```diff
@@ -0,0 +1,60 @@
+  const [user, setUser] = useState("");
+  const [message, setMsg] = useState("");
+  const [resArr, updateArr] = useState([]);
+  const handleClick = async() => {
```
if we could jump one line here between states and this handler

davidjhlu committed:
changes to app

- styling and readability changes
- changed the scope of variables to outside of function + added a variable for API URL
